### ISSUE TITLE
Offer a way to prevent KeeFox attempting to connect to KeePass

### ISSUE THREAD

**Issue by donestefan** [luckyrat/KeeFox#315]
Hi, 
I have Keefox 1.4.2 installed on Firefox 31 on Win7.

In my case Firefox's Password Store (which holds the KPRPC Key) is protected by a Master Password. It also holds some other passwords (ouside of keepass) which is why I don't want to disble the Master Password.

However, whenever I start Firefox it will prompt me for the Master Password - even if don't intend to visit a password protected website. Dismissing the Popup will result in a new popup only seconds later. Over and over again.
Clicking "not now" when Keefox asks me to login to the database will not change the behaviour of firefox's Password manager. Also the "not now" button will appear only after i already dismissed several popups.
I guess this is caused by keepass constantly trying to access the RPC Key.
It's quite annoying to type a superlongandverysecretmasterpassword when all I want to do is a quick google search.

Can this be fixed?

**Comment by luckyrat** [comment]
I've changed the issue summary to describe a desired outcome, based on your most recent comment. Does it make sense to you?

It wouldn't be too hard to implement but would need to be carefully tested and it would take a while to get all the options and user interface developed. For something that can so fundamentally break KeeFox I'd look at the following as a minimum:
* A big red cross on the main KeeFox icon to indicate it is broken
* An option to reveal the "disable" functionality
* The "disable" functionality itself (e.g. a button on the main KeeFox panel to enable/disable)

Once that gets broken down into manageable tasks, there's a fair bit of time involved in getting this feature enabled so I'm not intending to work on it unless I hear more support for the idea. However, if someone feels strongly enough to implement a suitable solution themselves, I'd be willing to accept a pull request.

PS: When I say my Firefox is open for days I mean that I put my PC into hibernation when not in use so there's no power/noise issue.

**Comment by donestefan** [comment]
Yes, the new summary perfectly makes sense.
I'm curious to see if anyone else is interested in such a feature.
Thanks for your feedback so far!

**Comment by luckyrat** [comment]
I'm not realistically going to get around to implementing this. If anyone comes across it and has a burning desire to re-open the issue and supply a pull request then feel free.
